spec.html

```diff
@@ -107,6 +107,30 @@ <h1>Well-Known Intrinsic Objects</h1>
 <emu-clause id="sec-abstract-operations">
   <h1>Abstract Operations</h1>
 
+  <emu-clause id="sec-SetterThatIgnoresPrototypeProperties" type="abstract operation">
+    <h1>
+      SetterThatIgnoresPrototypeProperties (
+        _home_: an Object,
+        _p_: a property key,
+        _v_: an ECMAScript language value,
+      ): either a normal completion containing ~unused~ or a throw completion
+    </h1>
+    <dl class="header">
+    </dl>
+    <emu-alg>
+      1. Let _O_ be ? RequireObjectCoercible(*this* value).
```

Comment:
The `this value` needs to be passed as an argument to an abstract operation, because the term `this value` only applies to functions defined after [ch. 18](https://tc39.es/ecma262/#sec-ecmascript-standard-built-in-objects). 

(The definitions for `this value` and `NewTarget` in ch. 18 seem to be kind of broken, because they refer to [ch. 10.3](https://tc39.es/ecma262/#sec-built-in-function-objects), which no longer defines these terms. For example compare to [ES 2020](https://262.ecma-international.org/11.0/#sec-built-in-function-objects), where `this value` and `NewTarget` are still mentioned in the "Built-in Function Objects" text.)